aten/src/ATen/core/Formatting.h

```diff
@@ -6,9 +6,12 @@
 #include <ATen/core/Type.h>
 #include <iostream>
 
-namespace at {
 
+namespace c10 {
```

Comment:
Is there any reason this can't live with the actual implementation?

Reply:
it's kind of common to keep it separate (boost does the same for optional I/O) because iostream is a huge header to pull in. Otherwise, there's no strong reason though and I don't have a strong opinion on it. It would definitely be more convenient to have it with the implementation.